test/governance/utils/Votes.test.js

```diff
@@ -21,41 +27,56 @@ contract('Votes', function (accounts) {
 
   describe('performs voting operations', function () {
     beforeEach(async function () {
-      this.tx1 = await this.votes.mint(account1, 1);
-      this.tx2 = await this.votes.mint(account2, 1);
-      this.tx3 = await this.votes.mint(account3, 1);
+      this.txs = await Promise.all(Object.entries(amounts).map(kv => this.votes.mint(...kv)));
     });
 
     it('reverts if block number >= current block', async function () {
       await expectRevert(
-        this.votes.getPastTotalSupply(this.tx3.receipt.blockNumber + 1),
+        this.votes.getPastTotalSupply(this.txs.at(-1).receipt.blockNumber + 1),
```

Comment:
Tests are failing because `.at` is not available until Node 16. Ideally our tests should run on Node 14 which is not deprecated until May this year. This is why the CI is configured with Node 14.

We can fix this by installing the dependency `array.prototype.at` and adding `require('array.prototype.at/auto')` at the top of this file.